### PR TITLE
refactor: disable pkce requirement for traditional app

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

disable pkce requirement for traditional app

## Testing

Local tested with SDKs.

## Checklist

- [x] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [ ] This PR is not applicable for the checklist

### CODE REVIEW

**Review by simeng-li — APPROVED** [review]

File: packages/core/src/oidc/init.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ export default function initOidc(
   const {
     resources: { findResourceByIndicator },
     users: { findUserById },
+    applications: { findApplicationById },
```
Comment (xiaoyijun) [inline packages/core/src/oidc/init.ts:52]: Unused?

**Review by xiaoyijun — APPROVED** [review]

File: .changeset/purple-birds-retire.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@logto/core": minor
```
Comment (gao-sun) [inline .changeset/purple-birds-retire.md:2]: `patch`?

**Review by gao-sun — DISMISSED** [review]

**Review by charIeszhao — APPROVED** [review]

File: packages/core/src/oidc/init.ts (hunk 1 above)
Comment (wangsijie) [reply packages/core/src/oidc/init.ts]: updated

File: .changeset/purple-birds-retire.md (hunk 2 above)
Comment (wangsijie) [reply .changeset/purple-birds-retire.md]: updated

**Review by gao-sun — APPROVED** [review]

**Pull request merged**